src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramUniformFactory.cpp

```diff
@@ -689,8 +689,9 @@ class UViewportInfo : public UniformGroup
 
 	void update(bool _force) override
 	{
+		float ySign = !GBI.isNegativeY() ? 1.0 : -1.0;
```

Comment:
conversion from double to float.

`float ySign = GBI.isNegativeY() ? -1.0f : 1.0f;`

Reply:
Done.